web-api-shared/src/elastic.rs

```diff
@@ -283,6 +283,7 @@ impl Client {
             return Ok(HashMap::new());
         }
         body.insert("_source".into(), json!(false));
+        body.insert("track_total_hits".into(), json!(false));
```

Comment:
❓ the es docs only list `track_total_hits` as a query param, but not as a body key. does this still has the desired effect and is not just silently discarded?

Reply:
Which documentation?

Like in the documentation linked in the issue ([track_total_hits](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-your-data.html) it's a body parameter :thinking:

Reply:
i see, i was referring to the [docs of the _search endpoint](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-search.html). looks like they forgot to update their docs properly.